heatpump.yaml

```diff
@@ -18,6 +18,10 @@ globals:
     type: uint16_t
     restore_value: no
     initial_value: '0'
+  - id: unmasked_value_register_210
+    type: uint16_t
+    restore_value: true
+    initial_value: '0'    
```

Comment:
Two questions: For `unmasked_value_register_0` the `initial_value` is comment out (we should remove it in an other pull request if not needed anymore), for the others `initial_value` is set.

1. Do we really need the `initial_value` ?
2. Now it is a string, although the type is set to `uint16_t`. Should this not be a number, thus without the quotes ?

Reply:
Hi 
In my oppinion initial_value can be exist -to avoid weird vsalues on startup.
And yes in specification int values are as string input in global -esphome ;)
I've removed restore_value to false -every reading from heat pump update this

Reply:
That was the reason indeed why a initial used that for the other globals. Then we should also set a `initial_value` for `unmasked_value_register_0`. I will create that change

Reply:
Ah, I see you already did. Ack !